packages/beacon-node/src/monitoring/service.ts

```diff
@@ -211,6 +208,16 @@ export class MonitoringService {
     }
   }
 
+  private nextMonitoringInterval(): void {
+    if (this.status === Status.Stopped) return;
```

Comment:
this check just prevents a theoretical edge due to a race condition case were `this.send()` has not yet finished but service was stopped. We would invoke `this.nextMonitoringInterval()` again and keep the interval running even though service is stopped.

This is probably never relevant if the BN or VC run as standalone services because the whole process will be stopped either way but if it is embedded in some another software this matters.

Reply:
Just noting here that this change is also quite important if BN or VC are standalone services as setting another timeout would prevent the process from exiting
- https://github.com/ChainSafe/lodestar/pull/5330